open-repositories.txt

```diff
@@ -18,6 +18,7 @@ linkedup
 motoko
 motoko-base
 motoko-playground
+portal
```

Comment:
Before we can accept exteranl contributions, this repo needs to be adapted to meet requirements outlined in go/opensource

Reply:
it has been approved in the Jira ticket